mmcls/evaluation/metrics/mce.py

```diff
@@ -0,0 +1,139 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import List, Optional, Sequence, Union
+
+import torch
+
+from mmcls.registry import METRICS
+from .single_label import Accuracy
+
+
+def get_ce_alexnet() -> dict:
+    """Returns Corruption Error values for AlexNet."""
+    ce_alexnet = dict()
+    ce_alexnet['gaussian_noise'] = 0.886428
+    ce_alexnet['shot_noise'] = 0.894468
+    ce_alexnet['impulse_noise'] = 0.922640
+    ce_alexnet['defocus_blur'] = 0.819880
+    ce_alexnet['glass_blur'] = 0.826268
+    ce_alexnet['motion_blur'] = 0.785948
+    ce_alexnet['zoom_blur'] = 0.798360
+    ce_alexnet['snow'] = 0.866816
+    ce_alexnet['frost'] = 0.826572
+    ce_alexnet['fog'] = 0.819324
+    ce_alexnet['brightness'] = 0.564592
+    ce_alexnet['contrast'] = 0.853204
+    ce_alexnet['elastic_transform'] = 0.646056
+    ce_alexnet['pixelate'] = 0.717840
+    ce_alexnet['jpeg_compression'] = 0.606500
+
+    return ce_alexnet
+
+
+@METRICS.register_module()
+class mCE(Accuracy):
```

Comment:
Better to use `CorruptionError` since `CE` is often used as the abbreviation of Cross Entropy. And remove the `mean` in the class name so that we can expand the class to output in different modes.